task2/src/main/java/backend/DuplicationNeuron.java

```diff
@@ -0,0 +1,18 @@
+package backend;
+
+public class DuplicationNeuron extends Neuron{
+    public DuplicationNeuron() {
+        super(1, x -> x);
+    }
+
+    @Override
+    public Double getTotalNeuronExcitation(double[] input) {
+        double totalNeuronExcitation = 0;
+
+        for (double v : input) {
+            totalNeuronExcitation += v;
+        }
+
+        return totalNeuronExcitation;
+    }
+}
```

Comment:
This removes weights from this type of Neuron, I think it's not correct

Reply:
That was the intention (duplication layer does not change anything) (ewentualne neurony wejściowe nie są zaliczane do neuronów przetwarzających)

Reply:
So in this case we can simplify it to 
```suggestion
package backend;

public class DuplicationNeuron extends Neuron {
    public DuplicationNeuron() {
        super(1, x -> x);
    }

    public Double getTotalNeuronExcitation(double input) {
        return input;
    }
}
```
or eaven remove input layer.